### PR TITLE
Add --extra-vars option for Ansible binding

### PR DESCRIPTION
**Pull request by matbu**

### CODE REVIEW

File: cmd/pull.go [hunk 1]
```diff
@@ -27,22 +27,28 @@ var cloud_engine string
...
 	environment or OCP. For example:
-  os-diff pull --cloud_engine=ocp --inventory=$PWD/hosts --output-dir=/tmp`,
+    os-diff pull --cloud_engine=ocp --inventory=$PWD/hosts --output-dir=/tmp`,
 	Run: func(cmd *cobra.Command, args []string) {
 
 		ansiblePlaybookConnectionOptions := &ansible.AnsiblePlaybookConnectionOptions{
 			Connection: "local",
 		}
 
+		envMap := make(map[string]interface{})
```
Comment (davidjpeacock) [inline cmd/pull.go]: Is this the best variable name for this?  Being verbose is better, I think?

**Review by davidjpeacock — CHANGES_REQUESTED** [review]
If you don't mind Mathieu, I'd love to see some better variable than envMap.

File: cmd/pull.go (hunk 1 above)
Comment (matbu) [reply cmd/pull.go]: ack I changed it in the next commit

**Review by davidjpeacock — APPROVED** [review]
LGTM - thanks!

**Pull request merged**